### PR TITLE
Update Dockerfile & CF ip addresses.

### PR DESCRIPTION
**Pull request by ghost**
I have updated the dockerfile and slimmed it down to 512mb (from over a gig) but it can still go farther. 

Also just pulled the updated list of CF IP addresses.

### CODE REVIEW

File: real-ip-from-cf
Comment (BobbyWibowo) [inline real-ip-from-cf]: Pretty sure these need the semicolons

edit: these reviews thing is confusing af

**Review by BobbyWibowo — CHANGES_REQUESTED** [review]

File: real-ip-from-cf
Comment (ghost) [reply real-ip-from-cf]: That is fixed now. Not sure if I can accept this review (don't see an intuitive way to do it), but it is in a new commit.

Comment (BobbyWibowo) [reply real-ip-from-cf]: Yeah, I noticed. I meant to mark this as resolved but I didn't understand how to use it, lmao

**Review by BobbyWibowo — APPROVED** [review]

File: real-ip-from-cf
Comment (BobbyWibowo) [reply real-ip-from-cf]: Ah got it. I didn't see the option when I was looking at it from mobile

**Comment by BobbyWibowo** [comment]
You may want to change maintainer line in the Dockerfile to yourself at this point. It was originally set as iCrawl because he was the one who made the Dockerfile in upstream (https://github.com/BobbyWibowo/lolisafe/commit/8a75ab91a6de2c963e390ea828daea235f0feac1#diff-3254677a7917c6c01f55212f86c57fbf). Upstream's actual dev is Pitu.
I don't know anything about Docker so I wouldn't want to claim credit either.

File: Dockerfile
Comment (BobbyWibowo) [inline Dockerfile:22]: Does this mean public visitors can access the lolisafe service through port 9999? If so, I don't think that's necessary. If using Nginx anyway, only Nginx should communicate locally with the service by port 9999 (which seems to be the case in `nginx.docker.conf` so that's good). Public visitors should always be proxied by Nginx through port 80 and 443.

**Comment by ghost** [comment]
I've changed it. Wasn't quite sure what best practice for this type of thing is. Do you have a version number? Upstream was at 3.0 2 years ago, so I've just left it at that.

**Comment by BobbyWibowo** [comment]
I've no idea either frankly. I guess it's fine as it is rn.

**Review by BobbyWibowo — CHANGES_REQUESTED** [review]

File: Dockerfile
Comment (BobbyWibowo) [reply Dockerfile:22]: I noticed you've recently changed this back to ready for review, mind sharing your thoughts about this? I think this is the only barrier left before merge.
Also I kinda ended up partially merging `real-ip-from-cf` cause I initially thought this was being drafted indefinitely.

**Comment by ghost** [comment]
No. This just tells other services, for example a different reverse proxy such as traefik what port to connect to. Opening sport looks like 'port: "9000:9000"'On Jul 2, 2020 3:24 AM, Bobby <notifications@github.com> wrote:
@BobbyWibowo requested changes on this pull request.



In Dockerfile:
>  
+RUN apk del build-dependencies
+
+COPY . .
+EXPOSE 9999

Does this mean public visitors can access the lolisafe service through port 9999? If so, I don't think that's necessary. If using Nginx anyway, only Nginx should communicate locally with the service by port 9999 (which seems to be the case in nginx.docker.conf so that's good). Public visitors should always be proxied by Nginx through port 80 and 443.

**Review by BobbyWibowo — APPROVED** [review]

**Comment by BobbyWibowo** [comment]
Merged in https://github.com/BobbyWibowo/lolisafe/commit/09d82ede4f86a2d3a738d2f936268701349d46e2.

**Pull request closed without merging**
